src/components/formatters/test/formatter_json_rpc_test.cc

```diff
@@ -40,12 +47,27 @@
 
 namespace test {
 namespace components {
-namespace formatters {
+namespace formatters_test {
 
 using namespace NsSmartDeviceLink::NsSmartObjects;
 using namespace NsSmartDeviceLink::NsJSONHandler::Formatters;
 using namespace NsSmartDeviceLink::NsJSONHandler::strings;
 
+void CompactJson(std::string& str) {
+  Json::Value root;
+  Json::Reader reader;
+  reader.parse(str, root);
+  Json::FastWriter writer;
+  str = writer.write(root);
+  std::string::iterator end_pos = std::remove(str.begin(), str.end(), '\n');
```

Comment:
This code removes all `\n` from the string. But what if this will be in the json string value? I assume that this will cause the corruption of the original string. This is not good. The logic should be next: just cut off last(search from the end of the string till first `}`) `\n` if exists. But since this code was already merged in the another repo, plus this is only for tests at the moment, lets leave as it is.